RELEASING.md

```diff
@@ -2,6 +2,9 @@
 
 _Instruction for Maintainers only._
 
+- Run [opentelemetry.io workflow](https://github.com/open-telemetry/opentelemetry.io/actions/workflows/build-dev.yml)
```

Comment:
```suggestion
- Run the [opentelemetry.io dev-build workflow](https://github.com/open-telemetry/opentelemetry.io/actions/workflows/build-dev.yml)
```

Reply:
The workflow accepts a regex as a argument:

> <img width="348" alt="image" src="https://github.com/user-attachments/assets/afdee88d-9f32-4b30-864e-e0f59c217f60" />

You can leave the argument as is with the default value, or change it to `opentelemetry-proto` as suggested in the next line (line 6) below.